README.md

```diff
@@ -3,6 +3,8 @@
 
 ## Setup
 
+- Copy local .m2/repository/ to ~/ProcessingService/m2/repository/
```

Comment:
It's not a good idea to ask prepare something manually, let's make it on the "install" mvn step for example.